packages/core/src/tracing/instrumentation/messaging/amqp.js

```diff
@@ -153,7 +153,6 @@ function instrumentedDispatchMessage(ctx, originalDispatchMessage, originalArgs)
   const consumerTag = fields.consumerTag;
   let consumer;
 
-  // 0.10.x
   // https://github.com/amqp-node/amqplib/commit/d483493e6825ed7d770e68f4cec60979d0feeae7
```

Comment:
NOTE: I was not able to remove the Node v10 check. The comment was wrong.

Reply:
I don't understand this comment. I think the `0.10.x` comment referred to the version number of the amqplib package, not to the Node.js version?

Reply:
hahahah yes you are right :D